R/util_qualtrics_cleaning.R

```diff
@@ -211,8 +221,14 @@ qc.rbind_inprogress <- function(inprogress_qdf, clean_qdf){
   # Qualtrics partial responses. However, they will appear once the partial
   # responses are closed and set to complete, and any survey elements that
   # depend on them (e.g, survey flow) should still work properly.
-
-  # First, just stop if the in-progress qdf and clean qdf have different
+  
+  # First, Qualtrics in-progress datasets automatically append an extra 
+  # column containing information about progress. This will break 
+  # rbinding, so append such a column to the clean_qdf.
+  
+  clean_qdf$partial_response_progress <- NA
```

Comment:
So other scripts were taking care of this lack-of-progress-column problem themselves before, right? I want to make sure this code gracefully handles that. The only problem case I can see is if those other scripts put information (not just NA) in there which you are overwriting with NA here. But I assume you guys never do that.

Reply:
I'm actually not sure what they were doing before. It would probably be a good idea to check on this.